src/Lookup/Lookup.tsx

```diff
@@ -962,6 +962,7 @@ export default class Lookup extends React.PureComponent<LookupProps, State> {
 					tabSelectsValue={false}
 					aria-label={ariaLabel}
 					aria-labelledby={ariaLabelledBy}
+					filterOption={() => true}
```

Comment:
filterOption is passed new callback on every re-render. Since this is a constant function you can declare it in the module scope or a static in the component class (it's up to you).

Reply:
added static function filterOption